qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/ManageUserPage.java

```diff
@@ -17,7 +17,7 @@
 	private static final By ADD_USER = By.linkText("Add User");
 	private static final By SAVE_BUTTON = By.id("saveButton");
 	private static final By HOME = By.id("homeNavLink");
-	private static final By ACTION = By.name("action");
+	private static final By ACTION = By.cssSelector("#content form table input[type=submit]");
```

Comment:
Can you use it like
```suggestion
	private static final By ACTION = By.cssSelector("#input[type=submit]");
```

Reply:
When i use `#input[type=submit]` as you have suggested, the element is lost but lemi trim it further to get `#content input[type=submit]` which is okay since it still points to the element

Reply:
why are we changing the element locator?

Reply:
The previous locator was so unstable. It was passing on the local instance but fail here on the pull request. So when I changed it, its now stable